software/bluesky-pds.yml

```diff
@@ -0,0 +1,14 @@
+name: "Bluesky PDS"
+website_url: "https://github.com/bluesky-social/pds"
+source_code_url: "https://github.com/bluesky-social/pds"
+description: "Personal data server to federate with the Bluesky social media network via AT Protocol."
+licenses:
+  - Apache-2.0
+  - MIT
+platforms:
+  - Docker
+  - Nodejs
```

Comment:
```suggestion
```
After reviewing the repo, shell seems to be the only primary method of installation, a dockerfile is present, but they don't explicitly make a guide for installing it via docker. feel free to correct collaborators or anyone. 👍🏻

Reply:
Ah, I think you are correct, nice catch. I saw node being used for the service, but it shouldn't be required on the system itself due to docker. Thanks!